src/themes/default/components/core/ProductGallery.vue

```diff
@@ -47,6 +49,32 @@
                     data-testid="productGalleryImage"
                     itemprop="image"
                   >
+                  <div
```

Comment:
all this video part should be a separate component which is displayed on v-if when image is video, to avoid to many logic in single component and make player reusable :)